package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ebay/ui-core-react",
-  "version": "3.7.0",
+  "version": "3.7.1",
```

Comment:
Minor issue, but how particular is the team about semver @abiyasa @darkwebdev ?  Technically this should be a minor version change, not a patch.

Reply:
updated accordingly to Henrique's conversation